src/main/java/seedu/address/model/util/OptionalDescription.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.util;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.Optional;
+
+/**
+ * Represents a Deliverable's description in the deliverable book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
+ */
+public class OptionalDescription {
```

Comment:
Add a note or TODO that this is to be renamed and Description is to be deprecated

Reply:
Could consider adding another static method / overloaded constructor that takes in a String so that we don't need to call Optional.of so many times